assets/js/components/settings/SettingsActiveModules.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * SettingsActiveModules component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { clearWebStorage } from '../../util';
+import { CORE_MODULES } from '../../googlesitekit/modules/datastore/constants';
+import Data from 'googlesitekit-data';
+import Layout from '../layout/Layout';
+import SettingsModule from './settings-module';
+import { Cell } from '../../material-components';
+const { useDispatch, useSelect } = Data;
+
+const SettingsActiveModules = ( { activeModule, moduleState, setModuleState } ) => {
+	const { submitChanges } = useDispatch( CORE_MODULES );
+	const [ error, setError ] = useState( false );
+	const [ isSaving, setIsSaving ] = useState( false );
+	const modulesData = useSelect( ( select ) => select( CORE_MODULES ).getModules() );
```

Comment:
Let's rename this for consistency; `modulesData` is how we name the legacy version of this.
```suggestion
	const modules = useSelect( ( select ) => select( CORE_MODULES ).getModules() );
```

Reply:
👍